src/OrleansAWSUtils/Streams/SQSAdapter.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Orleans.Streams;
+using OrleansAWSUtils.Storage;
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace OrleansAWSUtils.Streams
+{
+    internal class SQSAdapter : IQueueAdapter
+    {
+        protected readonly string DeploymentId;
+        protected readonly string DataConnectionString;
+        private readonly HashRingBasedStreamQueueMapper streamQueueMapper;
```

Comment:
Humm... In AzureQueueAdapter it uses the concrete type as well... Changed here to `IConsistentRingStreamQueueMapper`